index.html

```diff
@@ -258,10 +258,10 @@ <h2>Contact me</h2>
                     I’d love to help with it!</p>
             </header>
 
-            <form action="" id="contact-me-form">
-                <input type="text" name="user-name">
-                <input type="text" name="email">
-                <textarea name="message-box" id="message-box" cols="30" rows="8" placeholder="Write you message here..."></textarea>
+            <form action="https://formspree.io/f/mknayjll" id="contact-me-form" method="POST">
+                <input type="text" name="user-name" maxlength="30" placeholder="Full Name" required>
+                <input type="email" name="email" placeholder="Email address" required>
+                <textarea name="message-box" maxlength="500" id="message-box" cols="30" rows="8" placeholder="Write you message here..." required></textarea>
                 <button type="submit">Get in touch</button>
```

Comment:
- Hey, let's add the missing interaction states of each button in your webpage to indicate the (hover, disabled, enabled, active) to improve the user experience when the buttons are clicked:

![image](https://user-images.githubusercontent.com/59397159/235633008-a3613ba4-b895-4b39-89a7-af9eab9c12f5.png)